module-ballerina-openapi/Package.md

```diff
@@ -1,7 +1,9 @@
 ## Module overview
 
-The OpenAPI tools provide the following capabilities.
- 
+This package is for the Ballerina OpenAPI tooling which will make it easy for you to start the development of 
+a service documented in an OpenAPI contract in Ballerina by generating the Ballerina service and client skeletons. 
```

Comment:
Shouldn't we have a new line after the initial description? Otherwise whole paragraph will pick up as the description in the home page table? Isn't it @praneesha ?

Reply:
+1 Added.